data_steward/tools/clean_test_project.py

```diff
@@ -0,0 +1,109 @@
+# Python imports
```

Comment:
Nice refactor!  I have a question.  This is a relatively small standalone script.  It's a decent candidate for using the BigQuery client library instead of `bq_utils`.  Have you given any thought to that, or do you prefer `bq_utils`?

Reply:
Thanks! I haven't considered it yet. Do we have a consensus on which one to use? I can use it for my next task which is to identify datasets containing specific participants.

Reply:
This may be something to bring up at the standup Monday.  I don't think we want to mix and match inside modules, if we can avoid it.  But I personally think new development should start incorporating the client library when possible.  This is probably something we'll need to get consensus on.  If you're going to leave this with `bq_utils`, I can go ahead and approve it.